workflow/rocoto_funcs/base.py

```diff
@@ -67,7 +68,8 @@ def header_entities(xmlFile,expdir):
   text = f'''
 <!ENTITY ACCOUNT         "{account}">
 <!ENTITY QUEUE_DEFAULT   "{queue}">
-<!ENTITY PARTITION       "{partition}">'''
+<!ENTITY PARTITION       "{partition}">
+<!ENTITY CLUSTER         "{cluster}">'''
```

Comment:
"-M cluster" is specific to Gaea, we don't need to put it in the common_vars section.
So you may revert changes here. Thanks!